data/xml/Q18.xml

```diff
@@ -216,32 +215,32 @@
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1338/295</href>
-    <pages>225-240</pages>
+    <doi>10.1162/tacl_a_00017</doi>
+    <abstract>Neural encoder-decoder models of machine translation have achieved impressive results, while learning linguistic knowledge of both the source and target languages in an implicit end-to-end manner. We propose a framework in which our model begins learning syntax and translation interleaved, gradually putting more focus on translation. Using this approach, we achieve considerable improvements in terms of BLEU score on relatively large parallel corpus (WMT14 English to German) and a low-resource (WIT German to English) setup.</abstract>
+    <pages>225–240</pages>
     <url>http://www.aclweb.org/anthology/Q18-1017</url>
   </paper>
 
   <paper id="1018">
     <title>Questionable Answers in Question Answering Research: Reproducibility and Variability of Published Results</title>
     <author><first>Matt</first><last>Crane</last></author>
     <year>2018</year>
+    <doi>10.1162/tacl_a_00018</doi>
     <abstract>“Based on theoretical reasoning it has been suggested that the reliability of findings published in the scientific literature decreases with the popularity of a research field” (Pfeiffer and Hoffmann, 2009). As we know, deep learning is very popular and the ability to reproduce results is an important part of science. There is growing concern within the deep learning community about the reproducibility of results that are presented. In this paper we present a number of controllable, yet unreported, effects that can substantially change the effectiveness of a sample model, and thusly the reproducibility of those results. Through these environmental effects we show that the commonly held belief that distribution of source code is all that is needed for reproducibility is not enough. Source code without a reproducible environment does not mean anything at all. In addition the range of results produced from these effects can be larger than the majority of incremental improvement reported.</abstract>
-    <href>https://transacl.org/ojs/index.php/tacl/article/download/1299/296</href>
-    <pages>241-252</pages>
+    <pages>241–252</pages>
     <url>http://www.aclweb.org/anthology/Q18-1018</url>
     <video href="https://vimeo.com/276435177" tag="video"/>
   </paper>
 
   <paper id="1019">
     <title>Do latent tree learning models identify meaningful structure in sentences?</title>
     <author><first>Adina</first><last>Williams</last></author>
-    <author><first>Andrew</first><last>Drozdov</last></author>
+    <author><first>Andrew</first><last>Drozdov*</last></author>
```

Comment:
@aryamccarthy Do you know how this star got inserted?

Reply:
Oof. That definitely doesn't belong, @davidweichiang. The star is in the raw XML we're given by MIT Press; you can see that [they have it on their own site, too][link]. (It was a footnote mark to show Drozdov's new affiliation; should have been deleted upstream of us.)

[link]: https://www.mitpressjournals.org/doi/10.1162/tacl_a_00019